src/Lean/Data/JsonRpc.lean

```diff
@@ -37,7 +37,8 @@ inductive ErrorCode where
   | invalidRequest
   | methodNotFound
   | invalidParams
-  | internalError
+  | workerCrashed
```

Comment:
The `internalError` code is [a built-in](https://www.jsonrpc.org/specification#error_object) of JSON-RPC, so we should not change it. By convention we reserve the `326xy` range for those built-ins, so let's put the two new codes in the "Lean-specific" part, at `32901` and `32902`.